CONTRIBUTING.md

```diff
@@ -50,9 +50,10 @@ Every Issue and PR should be tagged with one or more of the following labels to
 
 <a name="prs"></a>
 ### Pull Requests
-- PRs must only be created when there is an issue present and a decision to create a PR is made. A PR must reference the issue it is resolving in the description field.
-  - using GitHub predefined keyword "**Fixes Issue#**" will automatically close the issue which the PR is resolving when it is merged.
-- A PR for an issues must only be created by the contributor assigned to the issue by the respective WSL.
+- PRs dealing with minor editorial changes can be raised without a corresponding Issue, but must be labelled as "Minor". Otherwise, PRs must only be created when there is an issue present and a decision to create a PR is made. 
+- A PR must reference the issue it is resolving in the description field.
+   - using GitHub predefined keyword "Fixes Issue#" will automatically close the issue which the PR is resolving when it is merged.
+- A PR for an issues must only be created by the contributor assigned to the issue (either by self assignment or by the respective WSL).
```

Comment:
Typo: "an issues" ---> "an issue"

Reply:
fixed in: https://github.com/cntt-n/CNTT/pull/1430/commits/ac19eb494314516098c02e77958e3461f66700d4